xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/variablesInit.vm

```diff
@@ -225,8 +226,8 @@
   ## currently, it comes from the old color theme object (see colorThemeInit.vm)
   // Default
   @xwiki-border-color:                  $theme.borderColor;
-  @xwiki-panel-header-bg:               $theme.panelHeaderBackgroundColor;
-  @xwiki-panel-header-text:             $theme.panelHeaderTextColor;
+  @panel-header-bg:                     $theme.panelHeaderBackgroundColor;
+  @panel-header-text:                   $theme.panelHeaderTextColor;
```

Comment:
Same here, wouldn't it be better to (additionally) keep the old variable names for backwards-compatibility?

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2497/commits/558749259766abd5e3cd5e8bfbf16fceed5bca47 :+1: